conftest.py

```diff
@@ -63,6 +63,18 @@ def dtype(dtype_name) -> torch.dtype:
     return TEST_DTYPES[dtype_name]
 
 
+@pytest.fixture(scope='session')
+def torch_optimizer():
+    try:
+        import torch._dynamo as dynamo
+
+        torch.set_float32_matmul_precision('high')
+        return dynamo.optimize('inductor')
```

Comment:
If we care with which backed is used here in tests, we can add the backend as an option (like dtype)

Reply:
Yeah, we can improve later